extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js

```diff
@@ -146,8 +149,12 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     /** @private {?../../amp-story/0.1/navigation-state.NavigationState} */
     this.navigationState_ = null;
 
-    /** @private {number} */
-    this.uniquePagesCount_ = 0;
+    /**
+     * Set to one here because we count on navigation event, and we do have
+     * an event before first page.
```

Comment:
i don't fully understand this. could you rephrase?

Reply:
I tried to explain above. Let me know if you think there is a better way to rephrase.

Reply:
Rewrote this. LMK if you think it should still be improved.